Scenarios/CustomizingALGoForGitHub.md

```diff
@@ -0,0 +1,282 @@
+# Customizing AL-Go for GitHub
+
+AL-Go for GitHub is a plug-and-play DevOps solution, intended to support 100% of the functionality needed by 90% of the people developing applications for Microsoft Dynamics 365 Business Central out-of-the-box.
+
+If AL-Go functionality out-of-the-box doesn't match your needs, you really have three options:
+
+1. Customize AL-Go for GitHub to fit your needs
```

Comment:
Could "Creating a feature suggestion in the AL-Go repo" be the first option? I guess we don't want too many people going down the path of customizing their AL-Go workflows

Reply:
very good idea